README.md

```diff
@@ -290,6 +291,7 @@ Options for the `Builder` class
     * `xmldec.standalone` standalone document declaration: true or false
   * `doctype` (default `null`): optional DTD. Eg. `{'ext': 'hello.dtd'}`
   * `headless` (default: `false`): omit the XML header. Added in 0.4.3.
+  * `cdata` (default: `false`): wrap text nodes in `<![CDATA[ ... ]]>` 
```

Comment:
> I guess checking the contents for these and only then soing CDATA if requested would work best.

The problem with that is we get into the sticky situations of what's valid and what's invalid XML? Crazy stuff like [ambiguous ampersands](https://mathiasbynens.be/notes/ambiguous-ampersands), etc.

I'm happy to do it, but I really don't know how to approach the problem (that is; I don't want to re-implement the entire [character data XML spec](http://www.w3.org/TR/REC-xml/#syntax) in this repo) - what are your thoughts?